modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -11,22 +11,16 @@ The saved states are listed by the ID of the user who made the change, time, dat
 
 Being able to view a saved state of a project is useful if you want to:
 
-* Undo an inadvertent change you made to a project by reverting back to a working state.
-* Review changes made by other users to the project.
-* Retrieve older values of some project configurations.
-
+* Undo an inadvertent change you made to a project and revert to the working state.
+* Review changes made by other users on the project.
+* Retrieve previous values of connector configurations.
 
 == Procedure
 
-To restore a previous state of your project:
-
-. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas. 
-. Click a saved state from the list.
-. Inspect the saved state in the sidebar by clicking on an object in the flow. Use this method to inspect all of the objects in the flow.
-. Continue either selecting different saved states from the list or the objects within them until you locate the saved state you want to restore.
-. Click *Restore* at the top of the canvas.
-. If you want to restore the selected saved state, click *Restore* in the dialog. 
-
-The restored state is renamed and moved to the top of the list.
-
+To restore to a previous version:
 
+. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas.
+. Click a saved state from the list. The saved state is a read-only view of each card in the project flow on the canvas.
+. Click on a card to view the configuration details of that object in the flow. Continue with this method to review any changes for a card or object in the flow.
```

Comment:
Perhaps line 24 could be "Click on any card to view the details of its configuration."?
About "Continue with this method": what method?
About "review any changes": are changes marked, so that users can see what changed?
About "for a card or object": What other objects besides cards? The first sentence in the step mentions only cards.
Both sentences end with "in the flow". Either use it in only one sentence or (better) drop it completely..

Reply:
Yes on 24. For this:
About "Continue with this method": what method? - what I was trying to say is that the user can continue to view the values of each card to see what's going on.  They aren't marked and there is no indication of any change. I have added some wording that perhaps fits. 
For this: About "for a card or object": What other objects besides cards? The first sentence in the step mentions only cards. - I'm still trying to solidify the terminology in my head. A card in the flow represents - a core component, an operation of a connector, a module, or an API. (Or so says the doc). Would object be a good overall term or do you suggest 
I specifically list these items? Are you saying that just using 'card' is not adequate here?
Removed flow.